diff -ruPN share/cubicweb/cubes/brainomics/views/actions.py ../catidb-trunk/share/cubicweb/cubes/brainomics/views/actions.py
--- share/cubicweb/cubes/brainomics/views/actions.py	2014-11-24 11:35:13.214097687 +0100
+++ ../catidb-trunk/share/cubicweb/cubes/brainomics/views/actions.py	2014-12-04 10:05:54.384898455 +0100
@@ -117,3 +117,6 @@
     # Desactivate some actions for now
     vreg.unregister(CopyAction)
     vreg.unregister(AddRelatedActions)
+
+def registration_callback(vreg):
+    pass
diff -ruPN share/cubicweb/cubes/brainomics/views/components.py ../catidb-trunk/share/cubicweb/cubes/brainomics/views/components.py
--- share/cubicweb/cubes/brainomics/views/components.py	2014-11-24 11:35:13.594099111 +0100
+++ ../catidb-trunk/share/cubicweb/cubes/brainomics/views/components.py	2014-12-04 10:05:46.172870066 +0100
@@ -335,8 +335,9 @@
     def layout_widgets(self, w, wdgs):
         super(BrainomicsFilterBox, self).layout_widgets(w, wdgs)
         w(u'<div class="admintools">')
-        for comp in self._cw.vreg['dynamic-facet'].possible_objects(self._cw, rset=self.cw_rset):
-            comp.render(w=w)
+        if 'dynamic-facet' in self._cw.vreg:
+            for comp in self._cw.vreg['dynamic-facet'].possible_objects(self._cw, rset=self.cw_rset):
+                comp.render(w=w)
         w(u'<div id="dynamic-facet-container"></div>')
         w(u'</div>')
 
@@ -400,3 +401,6 @@
     from cubicweb.web.views.basecomponents import ApplLogo, AnonUserStatusLink
     vreg.unregister(ApplLogo)
     vreg.unregister(AnonUserStatusLink)
+
+def registration_callback(vreg):
+    pass
diff -ruPN share/cubicweb/cubes/brainomics/views/primary.py ../catidb-trunk/share/cubicweb/cubes/brainomics/views/primary.py
--- share/cubicweb/cubes/brainomics/views/primary.py	2014-11-24 11:35:13.594099111 +0100
+++ ../catidb-trunk/share/cubicweb/cubes/brainomics/views/primary.py	2014-12-04 10:05:37.732840888 +0100
@@ -200,3 +200,6 @@
 def registration_callback(vreg):
     vreg.register_all(globals().values(), __name__, (BrainomicsPrimaryView,))
     vreg.register_and_replace(BrainomicsPrimaryView, PrimaryView)
+
+def registration_callback(vreg):
+    pass
diff -ruPN share/cubicweb/cubes/brainomics/views/secondary.py ../catidb-trunk/share/cubicweb/cubes/brainomics/views/secondary.py
--- share/cubicweb/cubes/brainomics/views/secondary.py	2014-11-24 11:35:13.594099111 +0100
+++ ../catidb-trunk/share/cubicweb/cubes/brainomics/views/secondary.py	2014-12-04 10:05:29.524812513 +0100
@@ -185,3 +185,5 @@
     vreg.register_and_replace(BrainomicsListView, ListView)
     vreg.register_and_replace(BrainomicsSameETypeListView, SameETypeListView)
 
+def registration_callback(vreg):
+    pass
diff -ruPN share/cubicweb/cubes/brainomics/views/startup.py ../catidb-trunk/share/cubicweb/cubes/brainomics/views/startup.py
--- share/cubicweb/cubes/brainomics/views/startup.py	2014-11-24 11:35:13.594099111 +0100
+++ ../catidb-trunk/share/cubicweb/cubes/brainomics/views/startup.py	2014-12-04 10:05:03.692723214 +0100
@@ -117,3 +117,6 @@
 
 def registration_callback(vreg):
     vreg.register_and_replace(BrainomicsIndexView, IndexView)
+
+def registration_callback(vreg):
+    pass
diff -ruPN share/cubicweb/cubes/brainomics/views/tabs.py ../catidb-trunk/share/cubicweb/cubes/brainomics/views/tabs.py
--- share/cubicweb/cubes/brainomics/views/tabs.py	2014-11-24 11:35:13.218097702 +0100
+++ ../catidb-trunk/share/cubicweb/cubes/brainomics/views/tabs.py	2014-12-04 10:05:14.080759151 +0100
@@ -232,3 +232,6 @@
                       (BrainomicsTabblableTimeline,))
     if 'vtimeline' in vreg.config.cubes():
         vreg.register(BrainomicsTabblableTimeline)
+
+def registration_callback(vreg):
+    pass
diff -ruPN share/cubicweb/cubes/brainomics/views/vtimeline.py ../catidb-trunk/share/cubicweb/cubes/brainomics/views/vtimeline.py
--- share/cubicweb/cubes/brainomics/views/vtimeline.py	2014-11-24 11:35:13.218097702 +0100
+++ ../catidb-trunk/share/cubicweb/cubes/brainomics/views/vtimeline.py	2014-12-04 10:00:09.699707200 +0100
@@ -93,6 +93,6 @@
             pass
 
 
-    def registration_callback(vreg):
-        vreg.register(VeriteCoTimelineJsonDataView)
-        vreg.register_and_replace(SubjectVeriteCoTimelineView, VeriteCoTimelineView)
+    #def registration_callback(vreg):
+        #vreg.register(VeriteCoTimelineJsonDataView)
+        #vreg.register_and_replace(SubjectVeriteCoTimelineView, VeriteCoTimelineView)
